src/validation.js

```diff
@@ -0,0 +1,16 @@
+export function isSeparatorsValid(separators) {
+  for (const separator of separators) {
+    if (!isNaN(separator)) return false;
+    if (separator === "-" || separator === ".") return false;
+  }
+  return true;
+}
+
+export function isExpressionValid(separator, expression) {
+  const expressionArr = expression.split("");
+  for (const str of expressionArr) {
```

Comment:
```suggestion
  for (const str of expression) {
```
배열로 변환하지 않아도 문자열이 이터러블해서 순회할 수 있을 것 같은데 어떻게 생각하시나요??

Reply:
0o0 왜... 왜그랬지? ㅋㅋㅋ 왜 생각을 못했을까요.. 감사합니다 이렇게 하는게 더 간결하고 좋아보이네요! 👍